src/mobile/src/main/java/tupperdate/android/appbars/ChatBars.kt

```diff
@@ -0,0 +1,78 @@
+package tupperdate.android.appbars
+
+import androidx.compose.foundation.Text
```

Comment:
Wrong `Text`. Should come from `material` package.
Probably due to the fact that we updated the compose version